cli/tools/test.rs

```diff
@@ -441,38 +450,35 @@ impl TestReporter for PrettyTestReporter {
 
   fn report_summary(&mut self, summary: &TestSummary, elapsed: &Duration) {
     if !summary.failures.is_empty() {
+      let mut failure_titles = vec![];
       println!("\nfailures:\n");
       for (description, js_error) in &summary.failures {
+        let mut failure_title = String::new();
+        failure_title.push_str(&description.name);
+        if let Some(location) = &description.location {
+          failure_title.push_str(
```

Comment:
When can it happen that there's no location?

Reply:
I guess there always has to be one because `Deno.test()` is sync... I'll make it non-optional.